pkg/skaffold/build/jib/errors.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jib
+
+import (
+	"fmt"
+
+	sErrors "github.com/GoogleContainerTools/skaffold/pkg/skaffold/errors"
+	"github.com/GoogleContainerTools/skaffold/proto"
+)
+
+func unknownPlugin(ws string) error {
```

Comment:
there are jib specific helper methods hence i have kept the plugin as is. 
The reason, why i decided to use specific error code "BUILD_UNKNOWN_JIB_PLUGIN_TYPE"  vs"BUILD_UNKNOWN_BUILDER_TYPE" as from the metrics data, its difficult to get chart on if it was thrown because jib plugin was not found or any other in future. We will have to find the builders when error code is "BUILD_UNKNOWN_BUILDER_TYPE"

Reply:
Maybe `unknownPluginType()`?